docs/advanced-config.md

```diff
@@ -89,10 +89,35 @@ export OTEL_PROPAGATORS=b3multi
 
 | System property          | Environment variable     | Default value  | Support | Description |
 | ------------------------ | ------------------------ | -------------- | ------- | ----------- |
-| `otel.traces.sampler`    | `OTEL_TRACE_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
+| `otel.traces.sampler`    | `OTEL_TRACES_SAMPLER`     | `always_on`    | Stable  | The sampler to use for tracing.	|
 
-Set `otel.traces.sampler` to `internal_root_off` to drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`. This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
+Splunk Distribution of OpenTelemetry Java supports all standard samplers as provided by 
+[OpenTelemetry Java SDK Autoconfigure](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#sampler).
+In addition, we add the following two samplers.
 
+### internal_root_off
+Setting `otel.traces.sampler` to `internal_root_off` will drop all traces with root spans where `spanKind` is `INTERNAL`, `CLIENT` or `PRODUCER`.
+This setting only keeps root spans where `spanKind` is `SERVER` and `CONSUMER`.
```

Comment:
I really really think we should *NOT* document this as something for general use.

Reply:
`internal_root_off` is already present in our documentation. But I will talk to Ivo and Fabrio about this.

Reply:
Any news?

Reply:
Any news?

Reply:
Any news?

Reply:
Any news?

Reply:
Any news?